ui-v2/app/styles/core/typography.scss

```diff
@@ -36,10 +36,10 @@ h1,
 h2,
 %header-nav,
 %healthchecked-resource header span,
-%healthcheck-status dt,
+%healthcheck-output dt,
 %copy-button,
```

Comment:
Calling the old `healthcheck-status` (which used to be essentially the places to see healthcheck logs) was making this confusing, so it was renamed `output`, `status` is now used for the individual little healthcheck icons with the counts/totals